### ISSUE TITLE
Module: FFMPEG Queue Manager

### ISSUE THREAD

**Issue by Morrisai** [Jam3/tech-we-want-to-use#19]
A module that wraps FFMPEG CLI and allows us to send multiple jobs that would be queued for server-side rendering.

**Comment by jmckinnell** [comment]
Yes please, then I could stop doing the Jam3 Site video encodes.

**Comment by mikkoh** [comment]
This could be a queue manager for all processor intensive tasks
